src/IotjsDebugger.ts

```diff
@@ -79,7 +79,7 @@ class IotjsDebugSession extends DebugSession {
     response.body.supportsFunctionBreakpoints = true;
     response.body.supportsEvaluateForHovers = false;
     response.body.supportsStepBack = false;
-    response.body.supportsRestartRequest = false;
+    response.body.supportsRestartRequest = true;
     response.body.supportsDelayedStackTraceLoading = true;
```

Comment:
I'm thinking that this should not be activated yet.  In this case vscode will send a restart request to the adapter, but processing the restart command is only supported in JerryScript.  This can lead to misleading and unexpected behavior with IoT.js.

Reply:
It we don't set it to `true`, then we can't send a restart request from jerry either.

Reply:
Yes, I know. Actually, it will not evaluate on the IoT.js side for now, but the implementation can remain here. I think someone will start the iotjs side implementation in the near future.